lib/rubocop/options.rb

```diff
@@ -23,8 +23,14 @@ def parse(command_line_args)
 
       @validator.validate_compatibility
 
-      if @options[:stdin] && !args.one?
-        raise ArgumentError, '-s/--stdin requires exactly one path.'
+      if @options[:stdin]
+        if args.any?
```

Comment:
With this change the file name given to `--stdin` is now an argument to the option, so it doesn't go into `args`. I can add a comment clarifying this.